data/mods/Magiclysm/monsters/monsters.json

```
@@ -539,5 +539,70 @@
     "death_function": { "corpse_type": "NO_CORPSE", "message": "The %s melts away." },
     "path_settings": { "avoid_traps": true, "avoid_sharp": true },
     "flags": [ "SEES", "WARM", "BASHES", "DESTROYS", "NO_BREATHE", "PUSH_MON", "PUSH_VEH", "FIREY", "PATH_AVOID_DANGER_1" ]
+  },
+  {
+    "id": "mon_razor_butterflies",
+    "type": "MONSTER",
+    "name": { "str": "razor butterfly", "str_pl": "razor butterflies" },
+    "//": "Now only for shaman, todo: make them natural spawn, add harvest, etc",
+    "description": "Some unknown species of butterfly, they have clear white wings with some orange rune pattern, and a razor-sharp part on the edge.  They are pretty big, but still in natural limits.",
+    "default_faction": "magical_beast",
+    "volume": "12 ml",
+    "weight": "13 g",
+    "bodytype": "insect",
+    "hp": 15,
+    "speed": 170,
+    "symbol": "'",
+    "color": "white",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 5,
+    "melee_dice": 1,
+    "melee_dice_sides": 1,
+    "melee_cut": 15,
+    "dodge": 8,
+    "luminance": 0,
+    "armor_bash": 0,
+    "armor_cut": 0,
+    "armor_bullet": 0,
+    "vision_day": 14,
+    "vision_night": 14,
+    "special_attacks": [ { "type": "leap", "cooldown": 3, "max_range": 2, "allow_no_target": true }, { "id": "cut_throat" } ],
+    "material": [ "iflesh" ],
+    "death_function": { "message": "A %s dissolve.", "corpse_type": "NO_CORPSE" },
+    "flags": [ "SEES", "HEARS", "NOHEAD", "HARDTOSHOOT", "FLIES", "NO_BREATHE", "NOGIB", "ATTACKMON", "STUN_IMMUNE", "SWARMS" ],
+    "//1": "has no HIT_AND_RUN cause they start to be too dumb"
+  },
+  {
+    "id": "mon_shambling_mound",
+    "type": "MONSTER",
+    "name": "shambling mound",
+    "description": "This giant plant mess is one of the ancients magic species in the world.  They don't like to walk too much, but the number of nutrients .",
```

add acid damage or smth idk
idk how to make it sound nice
```suggestion
    "description": "A magical, ancient Drosera.  A thick tangle of sap covered tentacles, though barely movable, it is capable of great destruction",
```